package.json

```diff
@@ -61,6 +61,9 @@
     "extends": [
       "stylelint"
     ],
+    "rules": {
+      "no-shadow": "error"
```

Comment:
I was thinking about adding it to the shared config right away.

Reply:
OK, I'll open a new PR on <https://github.com/stylelint/eslint-config-stylelint>. 👍 
And, when a new version of `eslint-config-stylelint` will be released, I'll remove this config in `package.json`.

No problems?

Reply:
Sure!